### PR TITLE
Creating kipp forward survey view

### PR DESCRIPTION
**Pull request by cristinabaldor**
View for KIPP Forward survey results with join to alumni fields

**Code checks:**
1) Is your branch up to date with `main`? Update from `main` and resolve and merge conflicts before submitting.
2) Are you `JOIN`-ing to a subquery? Refactor as a `CTE`.
3) Do your CTEs significantly transform the data, or could they be refactored into simple `JOIN`s?
4) Will every `SELECT` column be used downstream? Remove superfluous columns.
5) Does every table `JOIN` introduce columns that are used downstream? Remove superfluous `JOIN`s.
6) Double check that your SQL conforms to the style guide.
   * All tables should be referenced in three-parts: `{database}.{schema}.{table}`
   * All columns sould be prefixed with a table alias: `t.column_name`
   * All keywords should be UPPERCASE; all identifiers should be `snake_case`
   * In the event an identifier shares a name with a keyword, surround it with [square brackets].
   * Spaces, not tabs.
    
**What is the purpose of this view?**
> *[extract|feed|clean-up|other] Brief explanation...*

### CODE REVIEW

File: tableau/v_tableau.kipp_forward_survey.sql [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+     ,s.survey_response_id
+
+     ,ROW_NUMBER() OVER (
+      PARTITION BY status_c,student_c
+      ORDER BY actual_end_date_c DESC) AS rn_latest
+
+FROM gabby.alumni.contact c
+JOIN gabby.alumni.enrollment_c e
+  ON c.salesforce_id_c = e.student_c
+JOIN gabby.surveygizmo.survey_detail s
```
Comment (cbini) [inline tableau/v_tableau.kipp_forward_survey.sql:35]: you can remove this JOIN from the CTE, you're already joining to the view below

File: tableau/v_tableau.surveys.kipp_forward_survey.sql [hunk 2]
```diff
@@ -0,0 +1,80 @@
+USE gabby
+GO
+
+--CREATE OR ALTER VIEW tableau.kipp_forward_survey
+
+WITH salesforce_data AS (
```
Comment (cbini) [inline tableau/v_tableau.surveys.kipp_forward_survey.sql:6]: I think you can remove this CTE altogether, refactor the JOINs below

File: tableau/v_tableau.surveys.kipp_forward_survey.sql [hunk 3]
```diff
@@ -0,0 +1,80 @@
+USE gabby
+GO
+
+--CREATE OR ALTER VIEW tableau.kipp_forward_survey
```
Comment (cbini) [inline tableau/v_tableau.surveys.kipp_forward_survey.sql:4]: uncomment

File: tableau/v_tableau.kipp_forward_survey.sql [hunk 4]
```diff
@@ -0,0 +1,81 @@
+USE gabby
+GO
+
+CREATE OR ALTER VIEW tableau.kipp_forward_survey AS
+
+WITH alumni_data AS (
+SELECT
```
Comment (cbini) [inline tableau/v_tableau.kipp_forward_survey.sql:7]: indent 2 spaces, keep first select col on same line

File: tableau/v_tableau.kipp_forward_survey.sql [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+     ,e.actual_end_date_c
+     ,e.major_c
+     ,e.status_c
+
+     ,s.survey_response_id
+
+     ,ROW_NUMBER() OVER (
+      PARTITION BY status_c,student_c
+      ORDER BY actual_end_date_c DESC) AS rn_latest
+
```
Comment (cbini) [inline tableau/v_tableau.kipp_forward_survey.sql:31]: no space here

File: tableau/v_tableau.kipp_forward_survey.sql [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+     ,e.type_c
+     ,e.start_date_c
+     ,e.actual_end_date_c
+     ,e.major_c
+     ,e.status_c
+
+     ,s.survey_response_id
+
+     ,ROW_NUMBER() OVER (
+      PARTITION BY status_c,student_c
```
Comment (cbini) [inline tableau/v_tableau.kipp_forward_survey.sql:29]: indent 2 spaces

**Review by cbini — CHANGES_REQUESTED** [review]

File: tableau/v_tableau.surveys.kipp_forward_survey.sql (hunk 2 above)
Comment (cristinabaldor) [reply tableau/v_tableau.surveys.kipp_forward_survey.sql:6]: Hey, this was an old/unfinished file, deleting in the next commit.

File: tableau/v_tableau.surveys.kipp_forward_survey.sql (hunk 3 above)
Comment (cristinabaldor) [reply tableau/v_tableau.surveys.kipp_forward_survey.sql:4]: Hey, this was an old/unfinished file, deleting in the next commit.

File: tableau/v_tableau.kipp_forward_survey.sql (hunk 1 above)
Comment (cristinabaldor) [reply tableau/v_tableau.kipp_forward_survey.sql:35]: the output in survey_details is long (one row per question answer). I made it a CTE to match alumni data on just on the answer to one question (Salesforce ID), and so I could filter on rn_latest=1. Is there a way to do that on the main select? Basically I need every record in survey details to have a Salesforce ID tied to it.

**Review by cbini — APPROVED** [review]

**Pull request merged**